File: 구현/1159.cpp
#include <bits/stdc++.h>
using namespace std;

int n, a[26];
string s, ret;
int main() {
    cin >> n;
    for (int i = 0; i < n; i++) {
        cin >> s;
        a[s[0] - 'a']++;
    }

    for (int i = 0; i < 26; i++) {
        if(a[i] >= 5) ret += i + 'a'; 
    }

    if (ret.size()) {
        cout << ret << "\n";
    } else {
        cout << "PREDAJA" << "\n"; 
    }
    return 0;
}

File: 구현/1159.js
const input = [
  '18',
  'babic',
  'keksic',
  'boric',
  'bukic',
  'sarmic',
  'balic',
  'kruzic',
  'hrenovkic',
  'beslic',
  'boksic',
  'krafnic',
  'pecivic',
  'klavirkovic',
  'kukumaric',
  'sunkic',
  'kolacic',
  'kovacic',
  'prijestolonasljednikovi'
];

const n = Number(input[0]);
const arr = Array(26).fill(0);
for (let i = 1; i <= n; i++) {
  const s = input[i].charCodeAt(0);
  arr[s - 'a'.charCodeAt()]++;
}

let ret = '';
for (let i = 0; i < 26; i++) {
  if (arr[i] >= 5) {
    ret += String.fromCharCode(i + 'a'.charCodeAt());
  }
}

console.log(ret ? ret : 'PREDAJA');